src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,58 @@
+@import "../utils/variables";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 706px;
```

Comment:
This 706 is like a magic number, it is not clear why is it set to 706px and if you change for instance dimensions of calendar-day things will break up.
You should calculate the width basing on your variables ;)

Reply:
i even had it in variables but never used it 🤦